third_party/terraform/utils/test_utils.go

```diff
@@ -39,6 +39,11 @@ func (d *ResourceDataMock) GetOk(key string) (interface{}, bool) {
 	}
 }
 
+func (d *ResourceDiffMock) GetOk(key string) (interface{}, bool) {
```

Comment:
Make sure to put this method with the corresponding methods for the correct type.